homeassistant/components/sisyphus/__init__.py

```diff
@@ -35,7 +35,7 @@ class SocketIONoiseFilter(logging.Filter):
         """Filters out excessively verbose logs from SocketIO."""
 
         def filter(self, record):
-            if record.msg.contains("waiting for connection"):
+            if "waiting for connection" in record.msg:
```

Comment:
This change should not be included in this PR. Please fix this in another PR.

Reply:
Snuck in there from being based on another PR that's already merged. Will clean up.